src/models/information/_filters.js

```diff
@@ -75,6 +78,24 @@ const URBAN_ZONES = [
   "10-17",
 ]
 
+const provinceFilter = function() {
+  return provinces.reduce(
+    (acc, province) => ({
+      ...acc,
+      [province.name]: {
+        name: {
+          th: province.name,
+          en: province.name,
+        },
+        criterion: function(province, zone) {
+          return province.id === getProvinceByName(this.name.th).id
```

Comment:
I think you can use `province.id` but from outside.

Reply:
I created `getProvinceByName` because I misunderstand about how `criterion` method work, I will delete it, Thank you krubb.